components/amorphic/lib/startApplication.js

```diff
@@ -35,8 +35,19 @@ function startApplication(appName, appDirectory, appList, configStore, sessionSt
     if (config.serverMode === 'daemon' || config.serverMode === 'api') {
         controllerJsDir = path + '/js/';
     }
-
-    let schema = JSON.parse((readFile(path + '/schema.json') || readFile(commonPath + '/schema.json')).toString());
+    
+    let schema = null;
+    if (externalSchemas) {
+        schema = externalSchemas;
+    }
```

Comment:
copy pasta?  should this be a method / function

Reply:
Mike - one scenario is using common folder as well and in other scenario, we are only loading from the path provided. So I left the code as it's added the injection logic.

Reply:
Refactored the code